src/command_modules/azure-cli-keyvault/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+unreleased
+++++++++++
+* Update azure-keyvault SDK to 0.3.7
```

Comment:
Unless this triggers customer-facing changes, you don't need to mention it here. If it does, you should call those out instead.

Reply:
Re-reading the PR text, you should instead call out here that you fixed an issue where KV commands didn't work with ADFS.